<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGameWindow</class>
 <widget class="QMainWindow" name="MainGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainGameWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(205, 249, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="timeBar">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>11</y>
      <width>561</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>39</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>时间：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hintBtn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(11, 255, 36, 161);</string>
    </property>
    <property name="text">
     <string>hints</string>
    </property>
   </widget>
   <widget class="QPushButton" name="robot_btn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(11, 255, 36, 161);</string>
    </property>
    <property name="text">
     <string>auto</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>game</string>
    </property>
    <addaction name="actionBasic"/>
    <addaction name="actionMedium"/>
    <addaction name="actionHard"/>
   </widget>
   <addaction name="menuGame"/>
  </widget>
  <action name="actionBasic">
   <property name="text">
    <string>basic</string>
   </property>
  </action>
  <action name="actionMedium">
   <property name="text">
    <string>medium</string>
   </property>
  </action>
  <action name="actionHard">
   <property name="text">
    <string>hard</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
